service/settings.go

```diff
@@ -40,8 +40,8 @@ type svcSettings struct {
 	AsyncErrorChannel chan error
 }
 
-// AppSettings holds configuration for creating a new Application.
-type AppSettings struct {
+// ColSettings holds configuration for creating a new Collector.
+type ColSettings struct {
```

Comment:
Can we call this CollectorSettings? ColSettings is rather harder to associate with the collector because Col is not a common shorten name for collector unlike App is for Appllication.